packages/core/src/data-editor/data-editor.tsx

```diff
@@ -2518,14 +2518,14 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
         let [col, row] = args.location;
         const visible = visibleRegionRef.current;
         if (xDir === -1) {
-            col = visible.x;
+            col = visible.extras?.freezeRegion?.x ?? visible.x;
```

Comment:
This lets us account for forzen columns correctly.

Reply:
Makes sense; if you hover over frozen columns, you'd expect selection to also include it too.